docs/quick_start.md

```diff
@@ -87,19 +245,27 @@ Virtually Updating 'prod' ━━━━━━━━━━━━━━━━━━
 The target environment has been updated successfully
 ```
 
+Lines 2-4 show the completion percentage and run time for each model (very fast in this simple example). Line 8 shows that the `prod` environment now points to the tables created during model execution.
+
 You've now created a new production environment with all of history backfilled.
 
 ### 2.2 Create a dev environment
-Now that you've created a production environment, it's time to create a development environment so that you can make changes without affecting production. Run `sqlmesh plan dev` to create a development environment called `dev`.
-
-Although the summary of changes is similar, by showing that you've added two new models to this environment, the prompt notes that no backfills are needed and you're only required to perform a Virtual Update. This is because SQLMesh is able to safely reuse the tables you've already backfilled. Enter `y` to perform the Virtual Update:
+Now that you've created a production environment, it's time to create a development environment so that you can modify models without affecting production. Run `sqlmesh plan dev` to create a development environment called `dev`:
 
-```bash
-(.env) [user@computer sqlmesh-example]$ sqlmesh plan dev
+```bash linenums="1"
+$ sqlmesh plan dev
 ======================================================================
 Successfully Ran 1 tests against duckdb
 ----------------------------------------------------------------------
 New environment `dev` will be created from `prod`
+Apply - Virtual Update [y/n]: 
+```
+
+The output does not list any added or modified models because `dev` is being created from the existing `prod` environment without modification. 
+
+Line 6 shows that when you apply the plan creating the `dev` environment will only involve a Virtual Update. This is because SQLMesh is able to safely reuse the tables you've already backfilled in the `prod` environment. Enter `y` and press `Enter` to perform the Virtual Update:
```

Comment:
```suggestion
Line 6 shows that when you apply the plan creating the `dev` environment, it will only involve a Virtual Update. This is because SQLMesh is able to safely reuse the tables you've already backfilled in the `prod` environment. Enter `y` and press `Enter` to perform the Virtual Update:
```